src/mudlet-lua/lua/geyser/GeyserLabel.lua

```diff
@@ -54,8 +54,13 @@ function Geyser.Label:echo(message, color, format)
   if not fs then
     fs = tostring(self.fontSize)
   end
+  if color == "nocolor" then
+    color = [[ style=" ]]
+  else
+    color = [[ style="color: ]] .. Geyser.Color.hex(self.fgColor) .. "; "
```

Comment:
Just a cosmetic detail but would it be clearer if the semi-colon is put in as `[[; ]]`..?